konan/konan.properties

```diff
@@ -200,6 +200,44 @@ abiSpecificLibraries.linux_arm32_hfp = \
     lib/arm-linux-gnueabihf \
     usr/lib/arm-linux-gnueabihf
 
+# FIRST Robotics Competition
+targetToolchain-macos_x64-frc_arm32.default = \
+  /usr/local/arm-frc-linux-gnueabi
```

Comment:
Normally, students download this from here: http://first.wpi.edu/FRC/roborio/toolchains/